schema/swagger.yml

```diff
@@ -213,7 +215,3 @@ definitions:
       error:
         type: string
         example: Request limit exceeded. Please try again later.
-
-# Added by API Auto Mocking Plugin
-host: virtserver.swaggerhub.com
-basePath: /LiskHQ/Lisk/1.0.18
```

Comment:
Line missing

Reply:
@MaciejBaj Its been added in upcoming PR. Also adding ES-Lint rule for it, so this never happens again.